### ISSUE TITLE
The node order of PRISM15 does not match with gmsh format.

### ISSUE THREAD

**Issue by lw4992** [libMesh/libmesh#770]
In gmsh_io.C, PRISM15 was defined as following, 

```
em.add_def(ElementDefinition(PRISM15, 13, 3, 15)); // TODO: what should be done with this on writing?
```

Maybe not correct. Let me fix it.

**Comment by jwpeterson** [comment]
Hi @lw4992, Any luck with this issue?

**Comment by jwpeterson** [comment]
Fixed by #1192.
